tests/clade/query.rs

```diff
@@ -0,0 +1,32 @@
+use crate::clade::*;
+
+#[tokio::test]
+async fn test_basic_select() -> Result<(), Box<dyn std::error::Error>> {
+    let (context, clade) = start_clade_server().await;
+    tokio::task::spawn(clade);
+
+    tokio::time::sleep(tokio::time::Duration::from_secs(1)).await;
```

Comment:
mmmmm is there no better way to wait until it's up and running? this can randomly fail if something is having a slow day